src/sqs/error-handlers.ts

```diff
@@ -24,12 +24,17 @@ const EXPONENTIAL_BACKOFF_MULTIPLIER = 3;
 export const jiraAndGitHubErrorsHandler : ErrorHandler<any> = async (error: JiraClientError | Octokit.HookError | OldRateLimitingError | RateLimitingError | Error,
 	context: Context<any>) : Promise<ErrorHandlingResult> => {
 
-	const maybeResult = maybeHandleNonFailureCase(error, context);
+	const maybeResult = maybeHandleNonFailureCase(error, context)
+		|| maybeHandleRateLimitingError(error)
```

Comment:
I had to change it here, because of test failure.

`maybeHandleNonFailureCase` was returning the unretryable result if the response code was 403. 

I had to move it after the rate limiting check, because rate limiting errors have "status" field which can be 403.